### PR TITLE
v1.12.0 - Form toggle and full screen popover components

### PR DESCRIPTION
**Pull request by sonnyprince**
- Addition of form toggles component and variations
screen shot 2018-07-25 at 12 45 32
screen shot 2018-07-25 at 12 45 17
screen shot 2018-07-25 at 12 45 24

- Addition of full screen pop over component
screen shot 2018-07-20 at 09 54 39

- version bumps in CHANGELOG.md and package.json

### CODE REVIEW

**Comment by DamianMullins** [comment]
@sonnyprince Could you add some screenshots to the PR description so that we can get an idea of what the changes look like, please?

**Comment by DamianMullins** [comment]
@sonnyprince Is this ready for review?

**Comment by ashleynolan** [comment]
@sonnyprince Same with this PR – if you can update and let us know which order to review in, can get these in ASAP

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.12.0
```
Comment (DamianMullins) [inline CHANGELOG.md]: Version bump to v1.18.0.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+------------------------------
+*July 25, 2018*
```
Comment (DamianMullins) [inline CHANGELOG.md]: Update date.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,7 +39,6 @@ v1.10.0
 ### Removed
 - `resolutions` value for `fozzie-colour-palette` no longer needed
 
-
```
Comment (DamianMullins) [inline CHANGELOG.md]: keep this space, please.

File: docs/src/templates/pages/styleguide/ui-components/popover.hbs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+docs: true
+
+navgroup: styleguide
+navsub: ui-components
+navactive: popover
+---
+<p>A full screen pop over used for filters on narrow devices (please use less than mid screen to test)</p>
+
+{{#>demo-block baseUrl=../baseUrl }}
+    <div style="height: 700px; width: 300px; transform: translate3d(0,0,0)">
```
Comment (DamianMullins) [inline docs/src/templates/pages/styleguide/ui-components/popover.hbs]: Add these inline styles into the docs CSS?

File: package.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
   "name": "je-global-component-library",
   "title": "Just Eat – Component Library",
   "description": "Component Library and Guidelines for all of Just Eat’s Front-End Platforms",
-  "version": "1.10.0",
+  "version": "1.12.0",
```
Comment (DamianMullins) [inline package.json]: Bump.

**Comment by ashleynolan** [comment]
@sonnyprince Can you rebase this with master – can get this reviewed and merged then today 😄

**Comment by ashleynolan** [comment]
@sonnyprince Bump 😉 Would be good to update this and get it merged in (with any updates from the latest searchweb changes)

**Comment by ashleynolan** [comment]
@sonny Are you going to be adding these components to the docs?

**Comment by sonnyprince** [comment]
@ashleynolan - probably not the popover, f-megaModal will remove the need for this component (it's in my backlog to start in the coming sprint) 😃

**Pull request closed without merging**